src/components/Filter/FilterInput.vue

```diff
@@ -179,6 +178,10 @@ export default {
       type: String,
       default: () => '',
     },
+    shouldTruncateTags: {
```

Comment:
This prop should allow limiting the tags to the `TagLimit` from outside, based on some criteria.